resolwe_bio/tools/star_sj_to_bed12.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+"""Recalculate from STAR SJ.out.tab file to BED12 format."""
+
+import argparse
+
+import numpy as np
+import pandas as pd
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('-f', '--sj_file', help="STAR SJ.out.tab output file")
+args = parser.parse_args()
+
+sj_file = pd.read_csv(args.sj_file, delimiter='\t', header=None)
+
+# BED12 consists of 12 columns:
+header = ['chromosome',
```

Comment:
Fix to::

    header = [
        'chromosome',
        'sj_start',
        'sj_end',
        'sj_name',
        'score',
        'strand',
        'thick_start',
        'thick_end',
        'item_rgb',
        'block_counts',
        'block_sizes',
        'block_starts',
    ]

Whay is this better? Because:

  - You can change order of items in list without the need to remove/paste brackets or commas
  - When you change the name of `header` to somethign else (i.e. `columns` (that have one character more)) you dont have to re-align all of the items in list
  - indentation is smaller and you have more space in case of lond list items